packages/nodes-base/nodes/Asana/Asana.node.ts

```diff
@@ -1625,6 +1757,111 @@ export class Asana implements INodeType {
 					},
 				],
 			},
+			// ----------------------------------
+			//         project:update
+			// ----------------------------------
+			{
+				displayName: 'Workspace',
+				name: 'workspace',
+				type: 'options',
+				typeOptions: {
+					loadOptionsMethod: 'getWorkspaces',
+				},
+				displayOptions: {
+					show: {
+						resource: [
+							'project',
+						],
+						operation: [
+							'update',
+						],
+					},
+				},
+				default: '',
+				description: 'The workspace to create the project in',
+			},
+			{
+				displayName: 'Project',
+				name: 'projectId',
+				type: 'options',
```

Comment:
Project is type options but does not have any options?

Reply:
Originally had it as getProjects but it is not as reliable as using a string so I switched to a string option